releasenotes/notes/crio-workloadmeta-collector-52aa605d1f316532.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Adds a dedicated CRI-O Workloadmeta collector, enabling metadata collection
```

Comment:
Should "Workloadmeta" be two separate words?

Reply:
Double check on past release notes, it should just be one word.